src/_tests_/CountryIndex.test.js

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import CountryIndex from "../pages/CountryIndex";
+import  mockCountries  from "../mockCountries";
+
+describe("<CountryIndex />", () => {
+  it("renders without crashing", () => {
+    render(
+      <BrowserRouter>
+        <CountryIndex />
+      </BrowserRouter>
+    );
+
+    const component = screen.getByTestId("countryindex");
```

Comment:
Valid. Just FYI testid queries are saved for a last resort since RTL is more concern with what the user can see...I recommend checking out the other ways to query for an element on the UI.
https://testing-library.com/docs/queries/about/